src/clock.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import './App.scss';
+
+type Props = {
+  name: string,
+};
+
+type State = {
+  date: Date,
+};
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    date: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      const date = new Date();
```

Comment:
why u create const date instead of use date in state?

Reply:
good point